nautobot_circuit_maintenance/__init__.py

```diff
@@ -35,7 +34,9 @@ def custom_fields_extension(sender, *, apps=global_apps, **kwargs):  # pylint: d
             "label": "Provider Parser for Circuit Maintenance plugin.",
         },
     ]:
-        field, _ = CustomField.objects.get_or_create(name=provider_cf_dict["name"], defaults=provider_cf_dict)
+        defaults = {**provider_cf_dict}
+        name = defaults.pop("name")
+        field, _ = CustomField.objects.get_or_create(label=name, defaults=defaults)
```

Comment:
what does `label` do?

Reply:
`name` has been removed from `CustomField`, and `key` is populated from `label`.

Documentation here should be updated: https://github.com/nautobot/nautobot/blob/next/nautobot/docs/development/apps/migration/model-updates/extras.md#update-customfield-computedfield-and-relationship